kuma/static/styles/components/banners/_common.scss

```diff
@@ -2,31 +2,60 @@
     position: fixed;
     bottom: 0;
     width: 100%;
-    height: 130px;
+    height: 190px;
     text-align: center;
     z-index: 99;
 
+    @media #{$mq-small-desktop-and-up} {
+        height: 130px;
+    }
+
+    .slab-text {
+        background-color: #222;
+        color: #fff;
+        padding: 0 10px 0 20px;
```

Comment:
Is this an intentional design to have different padding on the left and right? Does it make sense in the mobile layout where it is centered?  Also, does it need an RTL variant?

Reply:
It is in the same vein as here:
https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow#Values

I did not add quite as much left padding but wanted to keep the same general idea. We do not *need* RTL, but this is in `_common.scss` because future banners might.